patch-upstream.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-
+set -e
```

Comment:
this should remain here because in case of any problem in the patch commands, the script will fail.